app/models/location.rb

```diff
@@ -1,5 +1,19 @@
+# frozen_string_literal: true
 class Location < ActiveRecord::Base
-  EDIT_FIELDS = %w(address_id booking_location_uid hidden hours organisation phone title uid).freeze
+  EDIT_FIELDS = %w(
+    address_id
+    booking_location_uid
+    hidden
+    hours
+    organisation
+    phone
+    title
+    uid
+    extension
+    twilio_number
+  ).freeze
+  PHONE_NUMBER_REGEXP = /\A\+44\d{9,10}\z/
```

Comment:
This is duplicated in `CallCenter`. Should we put this in a single, canonical location?